app/api-v1/services/identityService.js

```diff
@@ -0,0 +1,29 @@
+const fetch = require('node-fetch')
+
+const { BadRequestError, InternalError } = require('../../utils/errors')
+
+const { IDENTITY_SERVICE_HOST, IDENTITY_SERVICE_PORT } = require('../../env')
+const URL_PREFIX = `http://${IDENTITY_SERVICE_HOST}:${IDENTITY_SERVICE_PORT}/v1`
```

Comment:
do we want `IDENTITY_SERVICE_MAJOR_VERSION` as an env too? If not I'll remove from what I'm working on for `dscp-api`

Reply:
I don't think so. A major version change would mean a code change by design so I don't think having it as an env is of any benefit